### PR TITLE
added docs/user_guide/CVA6_UG_CSR.adoc

### PR DESCRIPTION
**Pull request by RanjanThales**
Created sub-directory docs/user_guide/ and added CVA6_UG_CSR.adoc into it

### CODE REVIEW

File: docs/user_guide/CVA6_UG_CSR.adoc [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+This is a chapter that will later be integrated in CVA6 users’ guide.
+
+== CSR in CVA6
+
+To anticipate future evolutions of the specification, for each custom CSR range defined in RISC-V specifications, it is recommended to:
+
+* Consider that the lower half is reserved for OpenHW Group (and not implement custom registers in this space).
+* Consider that the higher half is unused (i.e. left for custom variations of CVA6) and not use it in future CORE-V specifications.
+This is illustrated below for the 12'h7C0-12'h7FF range.
```
Comment (zarubaf) [inline docs/user_guide/CVA6_UG_CSR.adoc:21]:
```suggestion
This is illustrated below for the `12'h7C0`-`12'h7FF` range.

```

**Review by zarubaf — CHANGES_REQUESTED** [review]
Thanks a lot, that already looks very good. One additional request I have is to lower case the entire filename (also abbreviations like cva6 and csr). We just used lowercase letters for all files within the repository and we should stick to that (that requirement could/should probably also live in some doc :-))

File: docs/user_guide/CVA6_UG_CSR.adoc [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+|===
+|*NAME*|*ADDR*|*Description*|*Implemented*|*Mandatory CSR*|*Mode*
+
+|CSR_FFLAGS|12’h001|Floating point Accrued Exceptions|Yes|Yes|User
```
[the remark this answers is not in the document]
Comment (RanjanThales) [reply docs/user_guide/CVA6_UG_CSR.adoc:25]: Thanks Florian for your review and comments. I can work quickly on the part i.e. low case the entire filename and abbreviations. However, I didn't get you on the side of "mono spacing the code related things", could you elaborate on this !

[the remark this answers is not in the document]
Comment (zarubaf) [reply docs/user_guide/CVA6_UG_CSR.adoc:25]: Abbreviations can be upper case. Just the filename should be lower.

Mono-space means that you need to enclose the constants in ticks (``), that will format them in a mono-space font (just as it did in my comment above). See here: https://docs.asciidoctor.org/asciidoc/latest/text/monospace/

**Comment by RanjanThales** [comment]
Thanks Florian for the inputs, creating a updated PR with the modification.

**Pull request closed without merging**